packages/suite/src/services/github.ts

```diff
@@ -17,4 +19,49 @@ export const getReleaseNotes = async (version?: string) => {
     return release;
 };
 
+export const openGithubIssue = async () => {
+    const url = new URL(`${RELEASE_URL}/issues/new`);
+
+    let type = 'web';
+    if (isDesktop()) {
+        type = 'desktop app';
+    }
```

Comment:
General javascript audience would probably consider nicer not to create a new variable but interpolate inside the string instead, something like
```
`
Suite version: ${isDesktop() ? 'desktop app' : 'web' }
`
```
On the other hand, your code is more future proof if more platforms are to come and reuse this template :D as we don't like complicated or even nested ternaries.

Nevermind, I am just nitipicking :]

Reply:
Thanks! I actually wanted to do this and then forgot again that JS has the ternary operator (Python doesn't) :). I'll fix it. To educate myself: is there also a way to include `if`? Something like:

```
`
Suite version: ${if isDesktop(): something something }
`
```

Reply:
Actually the `?` symbol inside the ternary stands for `if`, but to some confusion being in post position instead of prefixing the condition. 
```
// long
if isDesktop()

// ternary
isDesktop() ? <expresion if true> : <expression if false>
```

Reply:
No no, sorry I know that. I meant if you can do some more complicated conditions inside the string. If you have for example also `for` available etc. But I guess no right? Because it is just a string a not a template mechanism?

Reply:
Haha, yeah, it thought that you wouln't be asking that. You can interpolate any expression there. So as for arrays you can do ['hello', 'world'].map(el => el.toUpperCase()).join(' '), Resulting in "HELLO WORLD". I doub't it would survive `for` clause, but I have never tried that :D.

Reply:
There is an interresting featature called template literals that gives you some additional powers https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals#Tagged_templates

Reply:
These templeta literals are behind the styled-components we use quite intensively https://medium.com/swlh/tagged-templates-and-understanding-styled-component-syntax-81b029da6ad

Reply:
Got it, thanks. Fixed in 926c3ac637d1d3c9bb07f0d669b6ccb068cda346.